packages/manager/apps/dedicated/client/app/dedicatedCloud/details/dedicatedCloud.routing.js

```diff
@@ -95,9 +95,12 @@ export default /* @ngInject */ ($stateProvider) => {
 
       managedVCDAvailability: /* @ngInject */ (ovhFeatureFlipping) =>
         ovhFeatureFlipping
-          .checkFeatureAvailability('hpc-vmware-managed-vcd')
-          .then((featureAvailability) =>
-            featureAvailability.isFeatureAvailable('hpc-vmware-managed-vcd'),
+          .checkFeatureAvailability(['hpc-vmware-managed-vcd', 'pcc-migration'])
+          .then(
+            (featureAvailability) =>
+              featureAvailability.isFeatureAvailable(
+                'hpc-vmware-managed-vcd',
+              ) && featureAvailability.isFeatureAvailable('pcc-migration'),
```

Comment:
As it's a sub feature, it should be like that rather
```suggestion
          .checkFeatureAvailability('hpc-vmware-managed-vcd:pcc-migration')
          .then(
            (featureAvailability) =>
               featureAvailability.isFeatureAvailable(hpc-vmware-managed-vcd:pcc-migration')),
```

Reply:
done